packages/angular_devkit/build_angular/src/webpack/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -100,13 +103,17 @@ export class CommonJsUsageWarnPlugin {
     });
   }
 
-  private hasCommonJsDependencies(dependencies: WebpackModule[], checkParentModules = false): boolean {
+  private hasCommonJsDependencies(
+    compilation: compilation.Compilation,
```

Comment:
Maybe just provide the `moduleGraph` instead of the entire `compilation`.

Reply:
`moduleGraph` it doesn't exist in wepback@4 as far as I know

Reply:
to expand my thought a little:
- if I provide an entire `compilation` object I can hide and inline types for webpack@5
- if I will try to provide only `moduleGraph`, invocation line will contain `as unknown as ModuleGraph`, 3 times in this file
So, I though 1st option is more clean

Reply:
SGTM